app/helpers/schema_helper.py

```diff
@@ -0,0 +1,46 @@
+class SchemaHelper(object):
+
+    @staticmethod
+    def get_first_group_id(survey_json):
+        return survey_json['groups'][0]['id']
+
+    @staticmethod
+    def get_last_group_id(survey_json):
+        return survey_json['groups'][-1]['id']
+
+    @staticmethod
+    def get_first_block_id(survey_json):
+        return survey_json['groups'][0]['blocks'][0]['id']
+
+    @staticmethod
+    def get_blocks(survey_json):
+        blocks = []
+        for group in survey_json['groups']:
+            for block in group['blocks']:
+                blocks.append(block)
+        return blocks
+
+    @staticmethod
+    def get_groups(survey_json):
```

Comment:
Worth considering if a generator would be better for these kinds of iterator/return a list methods, to avoid building up the whole list up-front and then iterating over it vs. iterating all elements one at a time when needed, which also allows you to quit the iterator before compiling the whole list if you find what you need earlier e.g.

```python
for group in survey_json['groups']:
     yield group
```
Usage
---
```python
for group in SchemaHelper.get_groups():
    ...

for index, group in enumerate(SchemaHelper.get_groups()):
    ...
```
Ref
---
- http://anandology.com/python-practice-book/iterators.html